app/components/workflow-assignment-dialog.jsx

```diff
@@ -2,7 +2,6 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Dialog from 'modal-form/dialog';
 import ReactDOM from 'react-dom';
-import { TextSplit } from 'seven-ten';
 import Translate from 'react-translate-component';
 
 class WorkflowAssignmentDialog extends React.Component {
```

Comment:
With `seven-ten` gone, I think we may be able to delete this entire component.

Reply:
Nope. This is used by Gravity Spy and informs volunteers when they have a new workflow assignment giving them an option to load the new workflow or continue on the current workflow.